### PR TITLE
Feature - search ui

### PR DESCRIPTION
**Pull request by at-lytruong**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Search/SearchViewController.swift [hunk 1]
```diff
@@ -10,7 +10,38 @@ import UIKit
...
+
+        searchs = [
+            Search(image: "image9", category: "Drink", name: "nuoc mia"),
+            Search(image: "image10", category: "Drink", name: "nuoc cam"),
+            Search(image: "image8", category: "Food", name: "com suon"),
+            Search(image: "image7", category: "Food", name: "bun cha ca"),
+            Search(image: "image6", category: "Food", name: "my quang"),
+            Search(image: "image5", category: "Food", name: "ngon com"),
+            Search(image: "image4", category: "Drink", name: "ngot nuoc")
+        ]
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Search/SearchViewController.swift]: Bỏ vào viewModel

File: MyApp/View/Controllers/Search/SearchViewController.swift [hunk 2]
```diff
@@ -10,7 +10,38 @@ import UIKit
...
+        UISearchBar.appearance().tintColor = UIColor.white
+        UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).tintColor = UIColor.blue
+
+        definesPresentationContext = true
+        searchController.dimsBackgroundDuringPresentation = false
+
+        searchController.searchBar.placeholder = "Search here...."
+        searchController.searchBar.scopeButtonTitles = ["All", "Foods", "Drinks"]
+        searchTableView.tableHeaderView = searchController.searchBar
     }
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Search/SearchViewController.swift]: Viết func config cho search

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**